edison/src/test/java/edu/illinois/cs/cogcomp/edison/annotators/SimpleGazetteerAnnotatorTest.java

```diff
@@ -211,26 +227,30 @@ public void testAddView() throws IOException, URISyntaxException, AnnotatorExcep
 	}
 
 	/**
-	 * Test method for {@link edu.illinois.cs.cogcomp.edison.annotators.SimpleGazetteerAnnotator#SimpleGazetteerAnnotator(java.lang.String)}.
+	 * Test method for {@link SimpleGazetteerAnnotator#SimpleGazetteerAnnotator(ResourceManager)}.
 	 * @throws URISyntaxException 
 	 * @throws IOException 
 	 */
 	@Test
 	public void testSimpleGazetteerAnnotatorString() throws IOException, URISyntaxException {
-		SimpleGazetteerAnnotator sga = new SimpleGazetteerAnnotator("/testgazetteers/");
+        Properties props = new Properties();
+        props.setProperty( SimpleGazetteerAnnotatorConfigurator.PATH_TO_DICTIONARIES.key, "/testgazetteers/" );
+        props.setProperty( SimpleGazetteerAnnotatorConfigurator.IS_LAZILY_INITIALIZED.key, SimpleGazetteerAnnotatorConfigurator.FALSE );
+        ResourceManager localRm = new SimpleGazetteerAnnotatorConfigurator().getConfig( new ResourceManager(props));
+		SimpleGazetteerAnnotator sga = new SimpleGazetteerAnnotator(localRm);
 		assertTrue ("Wrong number of dictionaries loaded.",sga.dictionaries.size() == 1);
 		assertTrue ("Wrong number of dictionaries loaded.",sga.dictionariesIgnoreCase.size() == 1);
 	}
 
-	/**
-	 * Test method for {@link edu.illinois.cs.cogcomp.edison.annotators.SimpleGazetteerAnnotator#SimpleGazetteerAnnotator(int, java.lang.String)}.
-	 * @throws URISyntaxException 
-	 * @throws IOException 
-	 */
-	@Test
-	public void testSimpleGazetteerAnnotatorIntString() throws IOException, URISyntaxException {
-		SimpleGazetteerAnnotator sga = new SimpleGazetteerAnnotator(6, "/testgazetteers/");
-		assertTrue ("Wrong number of dictionaries loaded.",sga.dictionaries.size() == 1);
-		assertTrue ("Wrong number of dictionaries loaded.",sga.dictionariesIgnoreCase.size() == 1);
-	}
+//	/**
```

Comment:
It should now be superfluous as the lazy init pattern as currently implemented requires a ResourceManager to be passed up to the Annotator, so different parameter configurations are accommodated by changing the RM entries.